examples/nextjs-blog/pages/blog/list.js

```diff
@@ -19,7 +19,11 @@ const Posts = function(props) {
 
       return (
         <div key={docId}>
-          <Link route={`/blog/${slug}`}>{`${title} - ${displayName}`}</Link>
+          <Link route={`/blog/${slug}`}>
+            {/* href gets added to <a> by <Link> */}
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+            <a>{`${title} - ${displayName}`}</a>
```

Comment:
Is the Link not adding an `<a>` tag already? Why add another one?

Reply:
the nextjs one does not do this, if you add a `<a>` it adds the href and then when you hover it is accessible (shows the url in the bottom of you screen like a normal link)

Reply:
that is very weird, but thanks for the info. Now the comment above makes more sense